WeakAuras/Transmission.lua

```diff
@@ -348,6 +348,12 @@ local function importPendingData()
       merged = {},
     }
   end
+  if data then
+    data.authorMode = nil
+  end
+  if oldData then
+    oldData.authorMode = nil
+  end
```

Comment:
If I have an aura, which I have in author mode, and I import an update to it, then because authorMode is considered as an internal field, WeakAuras.diff will not generate a patch for the authorMode field. So since the new data is actually discarded when there is both a patch and oldData, oldData needs its authorMode field cleared as well, or that aura will not be correctly set to userMode.